app/views/events/_applicants_overview.html.erb

```diff
@@ -41,18 +40,54 @@
-                  events_path, :class => 'btn btn-default'%>
+          events_path, :class => 'btn btn-default'%>
 
       <%= link_to "Alle drucken",
-                  events_path, :class => 'btn btn-default'%>
+          events_path, :class => 'btn btn-default'%>
     </div>
     <div class="btn-group pull-right" role="group">
-      <%= link_to "Zusagen verschicken",
-                  events_path, :class => 'btn btn-default'%>
-
-      <%= link_to "Absagen verschicken",
-                  events_path, :class => 'btn btn-default'%>
+      <% if (!@event.applications_classified?) %>
+          <div class="tooltip-wrapper disabled has-tooltip" data-toggle="tooltip" title="<%= t '.unclassified_applications_left' %>">
+            <%= button_tag t('.sending_acceptances'), type: 'button', class: 'send-emails-button btn btn-default', disabled: true %>
+          </div>
+          <div class="tooltip-wrapper disabled has-tooltip" data-toggle="tooltip" title="<%= t '.unclassified_applications_left' %>">
+            <%= button_tag t('.sending_rejections'), type: 'button', class: 'send-emails-button btn btn-default', disabled: true %>
+          </div>
+      <% elsif (@event.compute_free_places < 0)%>
+          <div class="tooltip-wrapper disabled has-tooltip" data-toggle="tooltip" title="<%= t '.maximum_number_of_participants_exeeded' %>">
+            <%= button_tag t('.sending_acceptances'), type: 'button', class: 'send-emails-button btn btn-default', disabled: true %>
+          </div>
+          <div class="tooltip-wrapper disabled has-tooltip" data-toggle="tooltip" title="<%= t '.maximum_number_of_participants_exeeded' %>">
+            <%= button_tag t('.sending_rejections'), type: 'button', class: 'send-emails-button btn btn-default', disabled: true %>
+          </div>
+      <% else %>
+          <%= button_tag t('.sending_acceptances'), type: 'button', class: 'btn btn-default', data: {toggle: 'modal', target: '#send-emails-modal', title: t('.sending_acceptances'), list: @event.accepted_applications_emails} %>
+          <%= button_tag t('.sending_rejections'), type: 'button', class: 'btn btn-default', data: {toggle: 'modal', target: '#send-emails-modal', title: t('.sending_rejections'), list: @event.rejected_applications_emails} %>
+      <% end %>
     </div>
   </div>
 
 </div>
+
+<!-- Bootstrap Modal for sending emails -->
+<div id="send-emails-modal" class="modal fade" role="dialog">
+  <div class="modal-dialog">
+
+    <!-- Modal content-->
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal">&times;</button>
+        <h4 class="modal-title">Emails verschicken</h4>
+      </div>
+      <div class="modal-body">
+        <%= link_to "Senden", '#', :id => 'send-emails-mailto', :class => 'btn btn-default' %>
+        <%= button_tag "In die Zwischenablage kopieren", type: 'button', id: 'send-emails-clipboard', class: 'btn btn-default' %>
```

Comment:
We are writing an email template in the current sprint and discard this modal. So i will leave it like this, because we will certainly need an own `locals.de.emails.yml` and remember this one!